citi-onboarding/tenis-center
# Service Card component

## Creation and implementing of the Service Card component.
* Title
* Description
* Image
* Gradient

## What I did

- Coding and Styling of the component
  - Implemented the 4 suggested props


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout Pedro`
   1. `git pull origin Pedro`
   1. `yarn install`
   1. `code .`
   1. `sudo docker-compose up`
   1. `yarn typeorm migration:run`
   1. `yarn start`
1. In your browser go to http://localhost:3000/<route>

1. In VScode open `<relative path>`

1. Things to do during the code:

   - Open insomnia and postbird to test the requisitions needed

1. Read the changed files section on GitHub

1. Test the things that you think are worth testing, even the ones that are not in this description

&nbsp;

---

**Time spent:** <see your time spent on https://wakatime.com/dashboard >
